settings.gradle

```diff
@@ -52,6 +52,10 @@ develocity {
                 tag jobName
             }
         }
+
+        capture {
+            fileFingerprints = true
+        }
```

Comment:
Why did you manually set this value? It seems the value is enabled by default.
https://docs.gradle.com/develocity/gradle-plugin/api/com/gradle/develocity/agent/gradle/scan/BuildScanCaptureConfiguration.html#getFileFingerprints()

Reply:
I believe I was migrating the old `taskInputFiles` parameter and found this one as the replacement. We can remove it if it is no longer needed.